test/Microsoft.ML.Tests/Scenarios/Api/AspirationalExamples.cs

```diff
@@ -60,5 +60,95 @@ public void FirstExperienceWithML()
                 PetalLength = 5.1f
             });
         }
+
+
+        [Fact]
+        public void SimpleIrisDescisionTrees()
+        {
+            var env = new ConsoleEnvironment();
+            string dataPath = "iris.txt";
+            // Create reader with specific schema.
+            var dataReader = TextLoader.CreateReader(env, ctx => (
+               label: ctx.LoadText(0),
+               sepalWidth: ctx.LoadFloat(1),
+               sepalLength: ctx.LoadFloat(2),
+               petalWidth: ctx.LoadFloat(3),
+               petalLength: ctx.LoadFloat(4)),
+               dataPath);
+
+
+            var pipeline = data.MakeEstimator()
+                .Append(row => (
+                    // Convert string label to key.
+                    label: row.label.Dictionarize(),
+                    // Concatenate all features into a vector.
+                    features: row.sepalWidth.ConcatWith(row.sepalLength, row.petalWidth, row.petalLength)))
+                .Append(row => (
+                    label: row.label,
+                    prediction: row.label.PredictWithMultiClassFastTree(row.features)));
+
+            // Load the data into the system.
+            var data = dataReader.Read(dataPath);
+            var model = pipeline.Fit(data);
+
+            var predictions = model.Transform(dataReader.Read(testDataPath));
+            var metrics = MultiClassEvaluator.Evaluate(predictions, row => row.label, row => row.prediction);
+        }
+
+        [Fact]
+        public void TwitterSentimentAnalysis()
+        {
+            var env = new ConsoleEnvironment();
+            var dataPath = "wikipedia-detox-250-line-data.tsv";
+            // Load the data into the system.
+            var dataReader = TextLoader.CreateReader(env, ctx => (
+                   label: ctx.LoadFloat(0),
+                   text: ctx.LoadText(1)),
+                   dataPath, hasHeader: true);
+
+            var pipeline = dataMakeEstimator()
+                .Append(row => (
+                    label: row.label,
+                    // Concatenate all features into a vector.
+                    features: row.text.TextFeaturizer()))
+                 .Append(row => (
+                    label: row.label,
+                    row.label.TrainLinearClassification(row.features)));
+
+
+            var (trainData, testData) = CrossValidator.TrainTestSplit(env, data: dataReader.Read(dataPath), trainFraction: 0.7);
```

Comment:
Why is `TrainTestSplit` coming from `CrossValidator`? This is a bit confusing.

Reply:
This is just a generic name which Pete suggest me to use.
I'm totally up to anything more reasonable, do you have a suggestion?

---
In reply to: [215109742](https://github.com/dotnet/machinelearning/pull/809#discussion_r215109742) [](ancestors = 215109742)

Reply:
Perhaps something like "`ModelValidation`"?
Scikit-Learn has [train_test_split()](http://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html) in model_selection

Reply:
I think we settled on extension to `DataView<TPig>`

---
In reply to: [215140323](https://github.com/dotnet/machinelearning/pull/809#discussion_r215140323) [](ancestors = 215140323)